### DISCUSSION TITLE
Sync status show count for bits of data - this number means nothing to the user

### DISCUSSION THREAD

**Discussion by jencastrodoesstuff** [digidem/mapeo-desktop#588] [Ideas]
**Describe the idea**
The numbers reflecting data transferred is not relevant to the user and can be confusing. 
"is that how many observations I have?"
If a reference number is given it should be # of new/edited observations
Possibly show somewhere how many observations one has on their computer as a reference point. 

**What's your role? Where and how do you currently use the application?**
Jen - Program Coordinator

**What is your ideal step by step workflow?**
just watch numbers

**Have you thought about what it might look like?**

**Additional context**

**Comment by okdistribute** [comment]
Would it be more immediately useful if this is a progress bar until we have a more full-featured way of showing syncronization status (like number of new/edited observations)? We probably would want to keep this progress bar long-term anyway
